playbooks/roles/mongodb/tasks/main.yml

```diff
@@ -135,5 +146,12 @@
       copy:
         content: "{{ MONGODB_EXPORTER_CONSUL_SERVICE | to_nice_json }}"
         dest: /etc/consul/mongodb-exporter.json
-      notify:
-        - reload consul
+      notify: reload consul
```

Comment:
Why change these lines?

Reply:
It was causing an issue that the corresponding handler is not found so I basically search the internet and found that If there is only a single item it should be written on the same line.

Reply:
@ahmedbilal You can comment this out during your testing, and revert the line back to it's previous value.

Reply:
@giovannicimolin Removed